tests/backup/backup_helper.go

```diff
@@ -5241,20 +5229,21 @@ func comparePodAge(oldPodAge map[string]nsPodAge) error {
 
 	allNamespaces, err := k8sCore.ListNamespaces(make(map[string]string))
 	for _, namespace := range allNamespaces.Items {
-		for _, skipCase := range namespacesToSkip {
-			if skipCase == namespace.GetName() {
-				pods, err := k8sCore.GetPods(namespace.ObjectMeta.GetName(), make(map[string]string))
-				if err != nil {
-					return fmt.Errorf("failed to get pods for namespace")
+		pods, err := k8sCore.GetPods(namespace.ObjectMeta.GetName(), make(map[string]string))
+		if err != nil {
+			return fmt.Errorf("failed to get pods for namespace")
+		}
+		log.InfoD("slice utils %v", sliceutil.Contains(namespacesToSkip, namespace.ObjectMeta.GetName()))
+		if sliceutil.Contains(namespacesToSkip, namespace.ObjectMeta.GetName()) {
```

Comment:
How about IsPresent?

Reply:
I felt it was better to use inbuil function instead. What's the advantage of using IsPresent() instead?

Reply:
I felt there is no need to import a new package.